### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by x-ivanova**
@lxdv

### CODE REVIEW

**Comment by lxdv** [comment]
Let's try to use some Markdown features. It would be useful as well. I think you could make a little list with useful links for html studying. Here is a list of features you **must** use in your README.md
- Emphasis (_italic_, **bold**)
- [Links](https://www.google.com)
- `Syntax Highlighting`
```javascript
var s = "JavaScript syntax highlighting";
alert(s);
```
- Tables

| Name | Age  | Sex  |
|---|---|---|
| Alexander  |  23 | M  |
| Nataly |  23 | F  |

- Blockquotes
> Hello

- Images
alt text

- You can also use HTML
```html
<dl>
  <dt>Definition list</dt>
  <dd>Is something people use sometimes.</dd>

  <dt>Markdown in HTML</dt>
  <dd>Does *not* work **very** well. Use HTML <em>tags</em>.</dd>
</dl>
```

<dl>
  <dt>Definition list</dt>
  <dd>Is something people use sometimes.</dd>

  <dt>Markdown in HTML</dt>
  <dd>Does *not* work **very** well. Use HTML <em>tags</em>.</dd>
</dl>

_It would be great if you could paste a video into README.md using inline HTML_

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# test-with-readme
\ No newline at end of file
+Alex this repo is for you <3
```
[the remark this answers is not in the document]
Comment (x-ivanova) [reply README.md]: Done. Check it please)

[the remark this answers is not in the document]
Comment (lxdv) [reply README.md]: Ok. Then do the task

File: README.md [hunk 2]
```diff
@@ -1 +1,28 @@
-# Alex this repo is for you <3
\ No newline at end of file
+# Alex this repo is for you <3
```
Comment (lxdv) [inline README.md]: Let's change this to Markdown practice

Comment (x-ivanova) [reply README.md]: Done

**Comment by lxdv** [comment]
OK. LGTM. Now you are able to merge this.

**Comment by x-ivanova** [comment]
@lxdv done

**Pull request merged**